Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,209 @@
+            if time_unit.startswith("minute"):
+              start_time = end_time - timedelta(minutes=int(number))
+            if time_unit.startswith("hour"):
+                dateobj = dateobj - timedelta(hours=int(number))
+            elif time_unit.startswith("day"):
+                dateobj = dateobj - timedelta(days=int(number))
+            elif time_unit.startswith("month"):
+                dateobj = dateobj - timedelta(days=int(number)*30)
+            elif time_unit.startswith("year"):
+                dateobj = dateobj - timedelta(days=int(number)*365)
+        except Exception:
+            return_error('invalid dateformat supplied')
+        return dateobj
+
+
+
+    def fetchincidents():
+        last_run=demisto.getLastRun()
+        if 'lt_time' not in last_run:
+            last_run['lt_time'] = parse_date_range(FETCH_TIME)
+        else:
+            last_run['lt_time']= datetime.datetime.strptime(last_run['lt_time'],'%d/%m/%Y %H:%M:%S.%f')
+
+        incidentids=get_incident_id(last_run['lt_time'])
+        incidents=[]
+        if len(incidentids['incidentId']) > 0:
+            for incident1 in incidentids['incidentId']:
+                inc=incidentDetail(str(incident1))
+                incidents.append({
+                    'name':"DLP Incident #"+str(incident1),
+                    'details':  inc[0]['incident']['violatedPolicyRule'][0]['ruleName']+ "RULE has been violated"
+                })
+                if inc[0]['incident']['incidentCreationDate'].isoformat() > last_run['lt_time'].isoformat():
+                    demisto.setLastRun({'lt_time':inc[0]['incident']['incidentCreationDate'].strftime("%d/%m/%Y %H:%M:%S.%f")})
+        demisto.incidents(incidents)
+        return(incidents)
+
+
+    def updateIncidents():
+        args = demisto.args()
+        inc_id = args['incidentId']
+        args.pop('incidentId')
+        if args['status'] == 'New':
+            args['status'] = 'incident.status.New'
+        return client.service.updateIncidents(updateBatch={'batchId':str(random.randint(11111,99999)),'incidentId':inc_id,'incidentAttributes':args})
+
+
+
+    def myconverter(o):
+        if isinstance(o, datetime.datetime):
+            return o.__str__()
+
+
+
+    if demisto.command() == 'fetch-incidents':
+        fetchincidents()
+    elif demisto.command() =='test-module':
+        try:
+            inc_list = get_incident_id(datetime.datetime.now())
+            demisto.results("ok")
+        except:
+            return_error("Invalid instance details")
+    elif demisto.command() == 'dlp-incedent-details':
+        inc_details = incidentDetail( demisto.args()['incidentId'])
+        res = helpers.serialize_object(inc_details[0])
+        demisto.results(json.loads(json.dumps(res,default = myconverter)))
+    elif demisto.command() == 'dlp-update-incident':
+        demisto.results(updateIncidents())
+
+  type: python
+  commands:
+  - name: dlp-incedent-details
```

Comment:
it seems that `dlp-incedent-details` returns incident details. So you need to add outputs to the command.
https://github.com/demisto/content/blob/master/docs/context_and_ouputs/README.MD here you have READE on outputs.
Outputs are important so that in the playbook, task B will be able to use the outputs of task A

Reply:
added the output to symantec.DLP.Incident